test/e2e_federation/service.go

```diff
@@ -53,6 +53,7 @@ var _ = framework.KubeDescribe("Federated Services [Feature:Federation]", func()
 	var _ = Describe("Without Clusters [NoCluster]", func() {
 		BeforeEach(func() {
 			fedframework.SkipUnlessFederated(f.ClientSet)
+			clusters = fedframework.ClusterSlice{}
```

Comment:
we should not do this. as my earlier comment. we should get actual clusters. like `clusters = f.GetRegisteredClusters()`